libp2p/Cargo.toml

```diff
@@ -85,6 +87,8 @@ uds = ["dep:libp2p-uds"]
 wasm-bindgen = ["futures-timer/wasm-bindgen", "instant/wasm-bindgen", "getrandom/js", "libp2p-swarm/wasm-bindgen"]
 wasm-ext = ["dep:libp2p-wasm-ext"]
 wasm-ext-websocket = ["wasm-ext", "libp2p-wasm-ext?/websocket"]
+websys = ["dep:libp2p-websys-transport"]
+websys-websocket = ["websys", "libp2p-websys-transport?/websocket"]
```

Comment:
I don't think we need a separate feature here or is there are a use of `websys` without activating this feature as well?

Reply:
If we ever make a WebTransport transport based on websys, I'd rather put that in a different crate than into here as a feature-flags.

Reply:
I added this as a feature following the discussion [here](https://github.com/vincev/libp2p-websys-transport/issues/5#issuecomment-1466957486) in case you want to implement a  WebRTC transport using[Websys](https://rustwasm.github.io/docs/wasm-bindgen/examples/webrtc_datachannel.html). 

You can use another crate to do that but you would have to maintain Websys  dependencies in two different places, I am not sure what you gain from that.

Reply:
Okay I see, shall we rename the folder then? Instead of websys websys-websocket.

Reply:
> Okay I see, shall we rename the folder then? Instead of websys websys-websocket.

Yep that sounds good to me!